modules/ROOT/pages/cross_cutting/oauth2_proxy.adoc

```diff
@@ -0,0 +1,70 @@
+:imagesdir: ../images
+= OAuth2 Proxy
+
+link:https://OAuth2-proxy.github.io/OAuth2-proxy/[OAuth2 proxy] is a link:https://en.wikipedia.org/wiki/Reverse_proxy[reverse proxy] that runs in front of your services to handle the complexity of link:https://developer.okta.com/blog/2019/10/21/illustrated-guide-to-OAuth-and-oidc[OpenID Connect / OAuth 2].
+Applications behind the proxy can be sure, that requests to them have already been authorized. 
+The proxy supports a variety of different link:https://en.wikipedia.org/wiki/Identity_provider[identity providers] like Google, GitHub, Keycloak...
+
+== Authentication flow
+
+A detailed flow diagramm can be found link:https://github.com/oauth2-proxy/oauth2-proxy/issues/1438[here]
+
+== Setup
+
+The OAuth proxy can be deployed from a link:https://github.com/OAuth2-proxy/OAuth2-proxy/releases/latest[prebuilt binary], a prebuilt link:https://quay.io/repository/OAuth2-proxy/OAuth2-proxy?tab=tags&tag=latest[docker image] or by link:https://github.com/OAuth2-proxy/manifests[helm chart] inside a Kubernetes cluster. It's recommended to use Kubernetes.
+
+image::OAuth2_Proxy-RequestFlow.drawio.svg[Request Flow]
+
+
+=== Keycloak 
+
+In this example, link:https://www.keycloak.org/[keycloak] is used as identity provider. For more details on keycloak see the devon link:https://devonfw.com/docs/java/current/cross_cutting/keycloak.html[documentation]. 
+To deploy keycloak in the cluster, it's recommended to use the helm chart from link:https://artifacthub.io/packages/helm/bitnami/keycloak[bitnami]. To configure the keycloak deployment, use a link:https://github.com/bitnami/charts/blob/main/bitnami/keycloak/values.yaml[values.yaml] file. After keycloak is running go to the keycloak admin ui and configure a client. Make sure to follow the link:https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider#keycloak-oidc-auth-provider[guide of OAuth2] proxy to set up the group mapping properly.
```

Comment:
Sadly I'm not a keycloak expert. But I'm having a strange feeling just recommending the bitnami helm charts without any validation process of those helm charts.

Maybe just remove it and also say that keycloak offers various deployment options. I would not recommend anything here.
The rest is fine